src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $color-grey;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    color: $color-main-accent;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
```

Comment:
margin-bottom?

Reply:
![image](https://user-images.githubusercontent.com/106028793/216449795-8f15103e-dd73-4046-acfb-51687b0fa01d.png)
I deleted it because lint told me to, I don't really get what the issue it is :(